exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/profile/ProfileResources.java

```diff
@@ -190,10 +192,13 @@ public QProfiles(List<ProfileInfo> runningQueries, List<ProfileInfo> finishedQue
     public List<String> getErrors() { return errors; }
   }
 
+  //max Param to cap listing of profiles
+  private static final String MAX_QPROFILES_PARAM = "max";
```

Comment:
It's just a convenient variable name for now, since it preserves the constant originally there `MAX_PROFILES`
The patch came as a result of this [exchange](https://www.mail-archive.com/user@drill.apache.org/msg05992.html) on drill-user list, so I didn't changing it for the patch. If we use pagination, perhaps then we can. 
For now, someone in the community has proposed this, which takes this further ahead: [DRILL-5068](https://issues.apache.org/jira/browse/DRILL-5068)